source/agg-pipeline-builder/pipeline-custom-collation.txt

```diff
@@ -0,0 +1,54 @@
+.. _pipeline-custom-collation:
+
+========================
+Specify Custom Collation
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use custom collation to specify language-specific rules for string comparison, 
+such as rules for letter case and accent marks.
+
+About This Task 
+---------------
+
+When entering a collation document, the ``locale`` field is mandatory. Default 
+collation parameter values vary depending on which locale you specify. To learn 
+more about supported languages and locales, see 
+:manual:`Collation Locales and Default Parameters </reference/collation-locales-defaults>`.
+
+
+Procedure
```

Comment:
[Task template]

I think according to our [Task Template](https://docs-mongodbcom-staging.corp.mongodb.com/docs/docsworker-xlarge/DOCSP-23914/information-types/task/) this heading should be "Steps".

Reply:
For this refactor, I've been mainly using "Procedure," but across Compass docs (and some server docs), I've noticed that we've been inconsistent with "Procedure" vs. "Steps." 

Procedure examples: 
- [Block Outgoing Network Connections ](https://www.mongodb.com/docs/compass/current/settings/restrict-outgoing-connections/#procedure)
- [Convert a Standalone to a Replica Set](https://www.mongodb.com/docs/manual/tutorial/convert-standalone-to-replica-set/#procedure)

Steps examples: 
- [Convert Replica Set to a Sharded Cluster](https://www.mongodb.com/docs/v7.0/tutorial/convert-replica-set-to-replicated-shard-cluster/#steps)
- [Create a Clustered Collection - Compass](https://www.mongodb.com/docs/compass/current/collections/clustered-collection/)

I'll bring this up in retro for clarity on what we _should_ be going with given how inconsistent it's been.

Reply:
Changing to "Steps" based off our discussion!